src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Staking withdrawals
+description: Page summarizing what staking push withdrawals are, how they work, and what stakers need to do to get their rewards
+lang: en
+template: staking
+image: ../../../assets/staking/leslie-withdrawal.png
+alt: Leslie the rhino with her staking rewards
+sidebarDepth: 2
+summaryPoints:
+  - The Shanghai upgrade brings staking withdrawals to Ethereum
+  - Validator operators must provide a withdrawal address to enable
+  - Rewards payments are automatically sent to the address provided every few days
```

Comment:
```suggestion
  - Rewards payments are automatically sent to the withdrawal address every few days
```

Reply:
I think we can go even a bit further and simplify it to

"Rewards are automatically distributed every few days"

Reply:
Given we mention the need for everyone to "provide a withdrawal address to enable" directly above, I agree we can simplify this. Agree with your adjustment Sam, but think Jakub's cleans it up nicely.

Reply:
```suggestion
  - Rewards are automatically distributed every few days
```